.github/workflows/build.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       matrix:
         suffix: ['', '-headless']
-        python: ['3.10', '3.9']
+        python: ['3.11', '3.10', '3.9']
```

Comment:
@jgerigmeyer should we start phasing out Python 3.9? The only active project using it is EE https://cs.github.com/?scopeName=All+repos&scope=&q=org%3Aoddbird+%22FROM+ghcr.io%2Foddbird%2Fpyjs%22

Reply:
Sure, fine with me -- what's the lift in upgrading EE?

Reply:
Hopefully changing that single line in the Dockerfile, running `pyupgrade`, and watching the tests. However, I just recalled that numpy or scikit-learn required an upgrade to support 3.10, so that might be blocking us. I'll add a card to keep track of that.